src/openvic-simulation/map/Map.cpp

```diff
@@ -529,21 +536,28 @@ bool Map::load_map_images(fs::path const& province_path, fs::path const& terrain
 					);
 				}
 			}
+
 			province_shape_image[idx].index = Province::NULL_INDEX;
 
-		set_terrain:
-			const TerrainTypeMapping::index_t terrain = terrain_data[idx];
-			TerrainTypeMapping const* mapping = terrain_type_manager.get_terrain_type_mapping_for(terrain);
-			if (mapping != nullptr) {
-				if (province_shape_image[idx].index != Province::NULL_INDEX) {
-					terrain_type_pixels_list[province_shape_image[idx].index - 1][&mapping->get_type()]++;
+			set_terrain:
```

Comment:
`set_terrain:` is a label, which is meant to be one indent level lower than the surrounding code. Can you move everything form line 542 to line 560 back one indent level?

Reply:
I highly doubt that. It was indented as if it was inside the for y loop, it's in the for x loop. It uses variables from the for x loop. So I indented it that way.

Reply:
The line containing `set_terrain` is a label, which goes one indent level lower than the code around it. All the code below stays at the same indent level as before.
With your changes, `set_terrain` is at the level of the x loop, and all the code below it is indented one level *more* than the x loop.

Reply:
That's very confused for me. Anyhow I unindented it.